Tests/Logic/ObjectMapping/RKPaginatorTest.m

```diff
@@ -218,9 +231,10 @@ - (void)testLoadingPageOfObjectMapsEntriesToObjects
     RKPaginator *paginator = [[RKPaginator alloc] initWithRequest:request paginationMapping:self.paginationMapping responseDescriptors:@[ self.responseDescriptor ]];
     [paginator loadPage:1];
     [paginator waitUntilFinished];
-    expect([paginator.mappingResult array]).will.haveCountOf(3);
+    // I cannot use here `haveCountOf` because prerequsities are checked not asynchronously and `nil` does not pass them
+    expect([[paginator.mappingResult array] count]).will.equal(3);
     NSArray *expectedNames = @[ @"Blake", @"Sarah", @"Colin" ];
-    expect([[paginator.mappingResult array] valueForKey:@"name"]).to.equal(expectedNames);
```

Comment:
It is because `RKObjectRequestOperation` calls both success and failure blocks as 

	dispatch_async(self.successCallbackQueue ?: dispatch_get_main_queue(), ^{
	    success(self, self.mappingResult);
	});

So it is called after test case tried to check it.